extensions/interactions/PencilCodeEditor/directives/pencil-code-editor-validation.service.spec.ts

```diff
@@ -86,7 +86,7 @@ describe('Pencil Code Editor Validation Service', () => {
       let rulesDict = rof.createNew('CodeEquals', inputBackend, {
         x: 'CodeString'
       });
-      let answergroup2 = agof.createNew([rulesDict], testOutcome2, [], '');
+      let answergroup2 = agof.createNew([rulesDict], testOutcome2, [], null);
```

Comment:
This function came from `AnswerGroupObjectFactory`.I didn't cover it in my last PR.This was a mistake on my end that I should also look for files that are somehow dependent on the files that I'm covering.Will keep this in mind!